### DISCUSSION TITLE
How can I enter the nether?

### DISCUSSION THREAD

**Discussion by HeroXp** [StoneLabs/strains-of-ascension#10] [Q&A] [answered]
It's been more than 2 hours and I can't access the nether, I've tried several times in different ways and I can't access it, does anyone know how to do it?

**Comment by StoneLabs** [comment, marked as answer]
<details>
<summary>Mild spoiler</summary>
| You have to be deep (like really deep) or have a "nether lead" (Artifact) in your inventory.
</details>

Be warned that the nether is insanely difficult in depth mode btw because i suck at balancing :). I plan on making it a bit easier in the future. If you find it too difficult use `gamerule doNetherStrains false`.
